lib/embed/media_tag.rb

```diff
@@ -49,7 +49,6 @@ def media_element(file, type)
             controls='controls'
             aria-labelledby="access-restricted-message-div-#{file_index}"
             class="sul-embed-media-file #{'sul-embed-many-media' if many_primary_files?}"
-            style="display:none; height: 100%;"
```

Comment:
The `mediaObject.show()` above was causing some elements (different on chrome vs safari) to be given a `display: inline` instead of `display: block` and the content was not visible.  It doesn't appear that we should need to hide/show the video or audio elements as this is handled up the DOM.